presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -481,6 +517,14 @@ public void testRound()
         assertFunction("round(TINYINT '3', 1)", TINYINT, (byte) 3);
         assertFunction("round(SMALLINT '3', SMALLINT '1')", SMALLINT, (short) 3);
         assertFunction("round(SMALLINT '3', 1)", SMALLINT, (short) 3);
+        assertFunction("round(FLOAT '3.0', 0)", FLOAT, 3.0f);
```

Comment:
It is testing 3 different scenarios, there is round(3.0) round(3.0, 0) and then round(3.0, 1)